### ISSUE TITLE
scheduling-review needs splitting up

### ISSUE THREAD

**Issue by dae** [ankitects/anki#2708]
https://forums.ankiweb.net/t/im-trying-to-improve-the-translation-but-i-cant-find-it-in-one-place-please-help-me/35335

**Comment by dae** [comment]
+ check whether AnkiWeb is treating old terms as a hard fail - the grace period has been long enough at this point.

**Comment by dae** [comment]
I've checked AnkiWeb and those messages are no longer in use. If terms haven't been confirmed, the sync will not be successful.

Regarding splitting the string, I think we should continue to use 'review' for both contexts in English, but use a separate key so that translators can alter it (and maybe add a comment explaining the context of each)

**Comment by xxhk-org** [comment]
This is also my feedback submission. I will add two more screenshots to illustrate this issue.

The meanings of the two positions in the image should be the same, right? 
  
However, the corresponding words are different. 
  
The English and Chinese on the right express the correct meaning, but the left side expresses a different meaning and is less easily understood than the right side.

<img width="1338" alt="Snipaste_2023-10-19_11-12-12" src="https://github.com/ankitects/anki/assets/9914772/9017840f-0b9f-4426-9293-c2c3b560c8ad">

<img width="1340" alt="Snipaste_2023-10-19_11-13-56" src="https://github.com/ankitects/anki/assets/9914772/4b1bce84-f5a1-4433-a2cc-d9553a40c7c6">

**Comment by dae** [comment]
The deck list uses a shorter wording, as the longer descriptions don't look good for table headings.

**Comment by xxhk-org** [comment]
> The deck list uses a shorter wording, as the longer descriptions don't look good for table headings.

Yes, I understand. 

What I mean is, just like before, by providing separate translation options, the translation on the deck screen can accurately convey the meaning without being confused with translations used elsewhere.

**Comment by abdnh** [comment]
"undo review" was removed in #2727 so it appears the change is no longer required?

**Comment by dae** [comment]
Well, that's convenient :-)

@xxhk-org sorry, I'm a little unclear about what you'd like changed in the deck list case. Could you please clarify where the same word is used in two different places where you want to use different wordings?

**Comment by xxhk-org** [comment]
> Well, that's convenient :-)
> 
> @xxhk-org sorry, I'm a little unclear about what you'd like changed in the deck list case. Could you please clarify where the same word is used in two different places where you want to use different wordings?

I mean here https://forums.ankiweb.net/t/im-trying-to-improve-the-translation-but-i-cant-find-it-in-one-place-please-help-me/35335?u=xxhk

**Comment by xxhk-org** [comment]
> Well, that's convenient :-)
> 
> @xxhk-org sorry, I'm a little unclear about what you'd like changed in the deck list case. Could you please clarify where the same word is used in two different places where you want to use different wordings?

Sorry, I may have misrepresented myself and the problematic translations are limited to the above screenshots.

**Comment by dae** [comment]
Sorry, I'm still not sure of exactly what you're asking us to change here. Could you please try rewording your request?

**Comment by xxhk-org** [comment]
> Sorry, I'm still not sure of exactly what you're asking us to change here. Could you please try rewording your request?

Sorry, I'll reorganize the language.

As you can see, the meanings of the left and right sides are the same, but the words and translations are different.
  
The right side indicates the state of "learning", but the left side is a verb, which doesn't express the state. I've reported this problem in the official forum before, and I hope that the left side "Learn" can be separated from the other "Learn" translations, so that it can be translated as "Learning"（学习中）, which is the Chinese equivalent of the state of being in the process of learning (including the two states of  Learning and Relearning).

<img width="802" alt="image" src="https://github.com/ankitects/anki/assets/9914772/c770e5b4-b26b-4e22-9386-16aea0248924">

**Comment by xxhk-org** [comment]
> Sorry, I'm still not sure of exactly what you're asking us to change here. Could you please try rewording your request?

<img width="1356" alt="image" src="https://github.com/ankitects/anki/assets/9914772/a35dc150-7a6b-4f94-a7ca-3ee280d1c184">

I tried the latest [23.10rc2](https://github.com/ankitects/anki/releases/tag/23.10rc2) version and found that the new translations are in the wrong place, the original **Deck** is fine and refers to the Deck list at the bottom, the new, standalone **Learn**（Learning） should be added in the second column as shown. 

And I didn't find where the new Learn is translated on [i18n](https://i18n.ankiweb.net/zh-CN/core/core/templates/decks.ftl/?string=24103).

**Comment by dae** [comment]
Well, that was silly of me. Thanks for letting me know. The new key is decks-learn-header
